supplementary_style_guide/glossary_terms_conventions/general_conventions/n.adoc

```diff
@@ -209,6 +209,18 @@
 
 *See also*: xref:vnic[vNIC], xref:smartnic[SmartNIC]
 
+[[nmstate]]
+==== image:images/yes.png[yes] Nmstate (noun)
```

Comment:
Just double checking that we're certain that "Nmstate" should be capitalized, even in the middle of a sentence? I Googled/looked through their sites, and it seemed to go both ways (Nmstate, nmstate).

Also, unfortunately, the "Kubernetes NMState Operator" seems to capitalize the N, M, and S. I don't think it's something we need to call out, just noticed when looking through the OpenShift docs for what we have in our repo. We have either that for the operator name, or lowercase "nmstate", but no just capital "N" :shrug:

Reply:
Before I submitted this entry, we had a discussion in the #nmstate Red Hat Slack channel. The product manager (Stanislas Faye) confirmed that the official spelling is "Nmstate", and the team will fix the wrong spellings on the upstream page.

The reason why I submitted this enhancement was because we have no consistent spelling in Red Hat docs. However, as an API that Red Hat uses in multiple of its products, it should be consistent.

Reply:
Thanks for the clarification!